docs/source/plugins/optional/varianter_yaml_to_mux.rst

```diff
@@ -140,8 +140,8 @@ The environment created for the nodes ``fedora`` and ``osx`` are:
 
 Note that due to different usage of key and values in environment we disabled
 the automatic value conversion for keys while keeping it enabled for values.
-This means that the value can be of any YAML supported value, eg. bool, None,
-list or custom type, while the key is always string.
+This means that the key is always string and the value can be of any YAML supported value, eg. bool, None,
+list or custom type, be aware that the None type will be passed to parameters as a string.
```

Comment:
This is best put as a separate sentence: "list or custom type. Please be aware that..."